graphql-java-kickstart/src/main/java/graphql/kickstart/execution/subscriptions/apollo/ApolloSubscriptionConsumer.java

```diff
@@ -23,7 +23,7 @@ public void accept(String request) {
       SubscriptionCommand command = commandProvider.getByType(message.getType());
       command.apply(session, message);
     } catch (JsonProcessingException e) {
-      log.error("Cannot read subscription command '{}'", request, e);
+      log.error("Cannot read subscription command '{}': {}", request, e);
```

Comment:
Why did you add the `: {}` part here? Since the way it was worked just fine letting it print the stack trace.